extensions/wikia/UserProfilePageV3/js/UserProfilePage.js

```diff
@@ -451,9 +451,8 @@ var UserProfilePage = {
 				userData[i] = userDataItem;
 			}
 		}
-		if (document.userData.hideEditsWikis.checked) {
-			userData.hideEditsWikis = 1;
-		}
+
+		userData.hideEditsWikis = (document.userData.hideEditsWikis.checked) ? 1 : 0;
```

Comment:
or you could use userData.hideEditsWikis = +document.userData.hideEditsWikis.checked;

Your call if it's readable or not, just pointing out an awesome javascript trick :)